frontend/aml_dashboard/src/index.js

```diff
@@ -0,0 +1,746 @@
+    });
+  })
+  .awaitPromises();
+
+const plotResultsNN = confidencePlot(predictionStreamNN);
+plotResultsNN.title = 'Results Neural Network';
+
+const plotResultsAML = confidencePlot(predictionStreamAML);
+plotResultsAML.title = 'Results AML';
+
+
+// -----------------------------------------------------------
+// INSTANCE VIEWER
+// -----------------------------------------------------------
+
+const create_fiware = button('Create');
+create_fiware.title = 'Fiware Node';
+
+// Configure the parameters for the Fiware Node
+const fiware_params = {
+  parameters: {
+    'Name': new Stream('MyFiwareNode', true),
+    'Domain': new Stream(0, true),
+    'Server IP': new Stream('192.168.1.51', true),
+    'Server Port': new Stream(1028, true),
+    'Context Broker IP': new Stream('localhost', true),
+    'Context Broker Port': new Stream(1026, true),
+    'Entity ID': new Stream('urn:ngsi-ld:Device:001', true),
+    'Attribute Data': new Stream('Data', true),
+    'Attribute Solution': new Stream('Inference', true),
+  },
+};
+
+const fiware_node = modelParameters(fiware_params);
+fiware_node.title = 'Choose parameters for the Fiware Node creation:';
+
+const fiware_node_status = text('Not Created');
+fiware_node_status.title = 'Fiware Node Status';
+
+const post_data = button('Post Data');
+post_data.title = 'Context Broker Data';
+
+const upload_data = imageUpload();
+upload_data.title = 'Upload data to the Context Broker';
+
+const data_status = text('Not Sended');
+data_status.title = 'Data Status';
+
+const get_solution = text('{}');
+get_solution.title = 'Context Broker Solution';
+
+const solution_status = text('Not Received');
+solution_status.title = 'Solution Status';
+
+let image;
+
+const url_context_broker = "http://localhost:5000/context_broker/";
+
+// Create the Fiware Node
+create_fiware.$click.subscribe( async () => {
+
+  fiware_node_status.$value.set(' <h1>Creating...</h1> <br> <img src="https://i.gifer.com/Cad.gif">');
+
+  const parameters = {};
+  for (let param in fiware_params.parameters) {
+    parameters[param] = fiware_params.parameters[param].value;
+  }
+
+  fetch(url_context_broker + 'fiware', {
+    method: "POST", // *GET, POST, PUT, DELETE, etc.
+    mode: "cors", // no-cors, *cors, same-origin
+    cache: "no-cache", // *default, no-cache, reload, force-cache, only-if-cached
+    credentials: "omit", // include, *same-origin, omit
+    headers: {
+      "Content-Type": "application/json",
+    },
+    redirect: "follow", // manual, *follow, error
+    referrerPolicy: "no-referrer", // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
+    body: JSON.stringify(parameters), // body data type must match "Content-Type" header
+  })
+  .then(response => response.json())
+  .then(async json => {
+    console.log(json);
+    console.log('CREATED');
+
+    fiware_node_status.$value.set('<h1>Created !</h1> ');
+
+  });
+});
```

Comment:
This is the code suggestion to add the option where the fiware node could not be created.
```suggestion
  try {
  const response = await fetch(url_context_broker + 'fiware', {
    method: "POST", // *GET, POST, PUT, DELETE, etc.
    mode: "cors", // no-cors, *cors, same-origin
    cache: "no-cache", // *default, no-cache, reload, force-cache, only-if-cached
    credentials: "omit", // include, *same-origin, omit
    headers: {
      "Content-Type": "application/json",
    },
    redirect: "follow", // manual, *follow, error
    referrerPolicy: "no-referrer", // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
    body: JSON.stringify(parameters), // body data type must match "Content-Type" header
  });
  if (!response.ok) {
    // If the response is not OK, display an error message
    fiware_node_status.$value.set('<h1> Fiware node could not be created</h1> ');
    console.error('Error creating FIWARE node:', response.status, response.statusText);
    return;
  }
   const json = await response.json();
   console.log(json);
   console.log('CREATED');

   fiware_node_status.$value.set('<h1>Created !</h1> ');

  } catch (error) {

    fiware_node_status.$value.set('<h1> Fiware node could not be created</h1> ');
    console.error('Network or other error:', error); }
  });
```